hunts/src/store.js

```diff
@@ -12,5 +18,13 @@ export default configureStore({
     puzzles: puzzlesReducer,
     hunt: huntReducer,
     tagFilter: tagFilterReducer,
+    filter: filterReducer,
```

Comment:
Is it easy to combine tagFilter and filter into one file? Logically they are all different kinds of puzzle filters. It seems we have 3 kinds right now: text filter, solved filter, and tag filter. I think it would be a little cleaner to have a single filterReducer that applies all 3.

(It would also be nice for localStorage to remember the last-applied tag filter, too.)

Reply:
done